lib/base.js

```diff
@@ -237,14 +166,14 @@ BaseElement = createClass({
    */
   convertAttributesToRefract: function(functionName) {
     var attributes = {};
-    var keys = _.keys(this.attributes);
+    var keys = this.attributes.keys();
 
     for (var i = 0; i < keys.length; i++) {
       var key = keys[i];
       if (this._attributeElementKeys.indexOf(key) !== -1) {
-        attributes[key] = this.attributes[key][functionName]();
+        attributes[key] = this.attributes.get(key)[functionName]();
```

Comment:
I've been wanting to discuss this code as well. Issue added https://github.com/refractproject/minim/issues/50